test/__mocks__/@testing-library/react.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { IntlProvider } from 'react-intl';
+import messages from '../../../i18n/locales/en.json';
+
+const {
+  render: rtlRender,
+  screen,
+  fireEvent,
+  waitFor,
+  // eslint-disable-next-line no-undef
+} = jest.requireActual('@testing-library/react');
```

Comment:
Why are you doing this instead of just importing testing-library normally?

Reply:
Be cause it's do like that on nodejs.dev.

> Personaly I never create a internationalized so I learn/discoverd I18n implementation now.